### DISCUSSION TITLE
formatting example for second

### DISCUSSION THREAD

**Discussion by mmehdidehghani** [atifaziz/NCrontab#98] [Q&A]
How can i set syntax for 10 second, I used"*/10 * * * * * and i got this error:
{"'*/10 * * * * *' is an invalid crontab expression. It must contain 5 components of a schedule in the sequence of minutes, hours, days, months, and days of week."}

**Comment by atifaziz** [comment]
```c#
var schedule = CrontabSchedule.Parse("*/10 * * * * *", new ParseOptions { IncludingSeconds = true });
```
